### PR TITLE
Added ppc6le build script for go-redis/redis

### PR DESCRIPTION
**Pull request by kumarvikas2020**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ? 

Note:
The Travis job will fail due to some test cases failing in parity with intel.
```# Observed 3 test failures for the version v8.0.0-beta.5, which are in parity with Intel.```

### CODE REVIEW

File: r/redis/redis_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : go-redis/redis
+# Version               : v8.0.0-beta.5
+# Source repo           : https://github.com/go-redis/redis.git
+# Tested on             : UBI 8.4
+# Language              : GO
```
Comment (seth-priya) [inline r/redis/redis_ubi_8.4.sh:7]: Please add the travis-check field

File: r/redis/redis_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+tar -C /usr/local -xzf ${GO_VERSION}.linux-ppc64le.tar.gz
+
+export GOROOT=/usr/local/go
+export GOPATH=$HOME/go
+export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+export GO111MODULE=auto
+fi
+
+# Download the repos
+git clone https://github.com/go-redis/redis.git
```
[the remark this answers is not in the document]
Comment (kumarvikas2020) [reply r/redis/redis_ubi_8.4.sh]: Script moved to go-redis directory.

File: r/redis/redis_ubi_8.4.sh (hunk 1 above)
Comment (kumarvikas2020) [reply r/redis/redis_ubi_8.4.sh]: travis-check field added.

File: g/go-redis/redis_ubi_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+else
+  go test -v ./...
+  ret=$?
+  if [ $ret -ne 0 ] ; then
+    echo "Tests failed "
+  else
+    echo "Build & unit tests Successful "
+  fi
+fi
+# Observed 3 test failures for the version v8.0.0-beta.5, which are in parity with Intel.
```
Comment (seth-priya) [inline g/go-redis/redis_ubi_8.4.sh:82]: Please specify details of the failing tests

Comment (kumarvikas2020) [reply g/go-redis/redis_ubi_8.4.sh]: Details are provided in the README.md file this has also been now mentioned in the build script.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**